src/hash.rs

```diff
@@ -0,0 +1,60 @@
+//! Keccak256 hash utilities.
+
+use tiny_keccak::{Hasher, Keccak};
```

Comment:
Do we really want to include this dependency just for hashing a single magic number? I'd be fine hardcoding it and commenting how it's derived.

Reply:
We need keccak hashing for signing and already had it as a dependency. I just moved the hashing to its own module so it was easier to reuse.

@fleupold I am not opposed to hard coding it and documenting how it is derived (and maybe add a unit test for good measure). Let me know if you prefer it that way.

Reply:
No strong feelings. Having this might come in handy some later time.